tests/Nhs.Appointments.Api.Integration/Scenarios/UserManagement/RemoveUserRoleAssignments.feature

```diff
@@ -0,0 +1,20 @@
+﻿Feature: Remove User roles from a Site
+
+  Scenario: Remove a user from a site
+    Given The following role assignments for 'test-one' exist
+      | Site | Roles                             |
+      | A    | canned:availability-manager       |
+    When I remove user 'test-one' from site 'A'
+    Then 'test-one' does not exist in the database
```

Comment:
Could we remove the implementation specific language (i.e. database) and perhaps change for then 'test-one' is no longer in the system